js/view/printView.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @param {jQuery object} container - references the HTML parent element that contains the view.
+ * @param {Object} model - the reference to the Dinner Model
+ */
+
+var PrintView = function (container, model) {
+    var numberOfGuests = container.find(".numberOfGuests");
+    var orderedItems = container.find("#orderedItemsList");
+
+    var loadOrderedItems = () => {
+        numberOfGuests.text(model.getNumberOfGuests());
+        model.selectedDishes.forEach(e => {
```

Comment:
Nice idea using e =>! However, maybe it is better to reserve 'e' for events. In this case 'e' actually is a dish, so maybe change it to 'dish' or 'd'?